src/RESTController.js

```diff
@@ -230,6 +232,11 @@ const RESTController = {
       }
     }
 
+    const transaction = options.transaction;
+    if (transaction) {
+        payload.transaction = true;
```

Comment:
You might want to lint your code.

Reply:
I am quite new to this and don't know what needs to be done exactly. Can you give me some advises how to get started?

Reply:
Just run `npm run lint` or `yarn lint` If you are using yarn

Reply:
Check out the [Contributing Guide](https://github.com/parse-community/Parse-SDK-JS/blob/master/CONTRIBUTING.md) for all your testing and linting needs.